Makefile

```diff
@@ -61,7 +61,7 @@ test-go-serial:
 
 .PHONY: test-integration
 test-integration:
-	go test ./integration-tests -timeout 185s --integration
+	go test ./integration-tests -timeout 185s --enable-browser-integration-tests
```

Comment:
Technically this means we are running the non-browser integration tests twice. Once in `test-go` and again here. How about we change this Make task so that it _only_ runs the browser tests?

```
.PHONY: test-browser-integration
test-browser-integration:
	go test ./integration-tests -timeout 185s --enable-browser-integration-tests -run BrowserIntegration
```

Then we also need to update __.circleci/config.yml__.